packages/manager/modules/pci/src/projects/project/private-networks/translations/Messages_fr_FR.json

```diff
@@ -3,9 +3,13 @@
   "pci_projects_project_network_private_error": "Une erreur est survenue lors du chargement des informations {{ message }}",
   "pci_projects_project_network_private_create": "Créer un réseau privé",
   "pci_projects_project_network_private_name": "Nom",
-  "pci_projects_project_network_private_id": "ID",
-  "pci_projects_project_network_private_region": "Localisations",
   "pci_projects_project_network_private_vlan_id": "VLAN ID",
-  "pci_projects_project_network_private_address": "Adresse de réseau privé (CIDR)",
+  "pci_projects_project_network_private_action": "Plus",
+  "pci_projects_project_network_private_region": "Région",
+  "pci_projects_project_network_private_address": "CIDR",
+  "pci_projects_project_network_private_gateway": "Passerelle",
+  "pci_projects_project_network_private_dhcp": "DHCP",
```

Comment:
**suggestion:** Can be maintained as constant since it cannot be translated.

Reply:
same here: https://github.com/ovh/manager/pull/7324#discussion_r893116095